src/main/java/me/zilzu/kiosk/owner/OwnerService.java

```diff
@@ -0,0 +1,18 @@
+package me.zilzu.kiosk.owner;
+
+import org.springframework.stereotype.Service;
+
+@Service("ownerService")
```

Comment:
```suggestion
@Service
```
라고만 써도 잘 바인딩됩니다.

같은 인터페이스에 대한 Bean을 여러개 만들려고할 때, 이런식으로 쓰는데요.
지금은 필요없어보입니다.